specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/examples/TrustedAccessRoles_List.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "api-version": "2022-04-02-preview",
+    "subscriptionId": "subid1",
+    "location": "westus2"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "sourceResourceType": "Microsoft.MachineLearningServices/workspaces",
+            "name": "reader",
+            "rules": [
+              {
+                "verbs": [
+                  "get"
+                ],
+                "apiGroups": [
+                  ""
```

Comment:
Does this mean all APIGroups? Or none?

Reply:
`""` indicates the core API group, e.g. pod